src/styles/blocks/calendar.scss

```
@@ -0,0 +1,57 @@
+@import "../utils/mixins";
+@import "../utils/veriables";
+@import "../utils/extands";
+
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: ($square-size * 7) + 10px;
+    margin-top: 21px;
+    gap: $gap-size;
+  }
+
+  &__day {
+    display: flex;
+    @include square($square-size);
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+
+    background-color: #eee;
+
+    @extend %date-text;
+    list-style: none;
+
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+
+      transform: translateY(-20px);
+      transition: 0.5s;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day--#{$i}::before {
+      content: "#{$i}";
+      position: absolute;
+    }
+  }
```

Add a number (Arial 30px) in the center of each day using ::before and [@for](https://sass-lang.com/documentation/at-rules/control/for), use nth-child here, not modifiers
```suggestion
  @for $i from 1 through 31 {
    &:nth-child(here you should use your index)::before {
      content: "#{$i}";
      position: absolute;
    }
  }
```